daemon/create.go

```diff
@@ -105,6 +105,11 @@ func (daemon *Daemon) containerCreate(ctx context.Context, opts createOpts) (con
 	}
 	containerActions.WithValues("create").UpdateSince(start)
 
+	const hostPortWarning = `Container is created with host port mappings which may bypass firewall rules and give access to the container from the outside world.`
```

Comment:
@akerouanton this doesn't just apply to UFW; it applies to all Linux firewalls due to the use of iptables. UFW, firewalld - they orchestrate the firewall usage for users. So while it would be nice for Docker's firewall rules to show up in those tools the bigger issue is that Docker bypasses the firewall in such a way as it breaks those tools and even breaks firewalls for those *not* using those tools (but writing firewall rules by hand).